src/i_winmusic.c

```diff
@@ -1567,7 +1567,10 @@ static boolean I_WIN_InitMusic(void)
 
     if (i == all_devices)
     {
-        free(winmm_midi_device);
+        if (strlen(winmm_midi_device) > 0)
```

Comment:
This could theoretically cause a memory leak if the user-specified midi device is an empty string.

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/pull/1642/commits/7929bbc56c036a0261b5d28bdf4782d0e7bb6098